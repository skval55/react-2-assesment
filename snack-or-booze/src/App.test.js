import { render, screen } from "@testing-library/react";
import App from "./App";
import React from "react";
import { MemoryRouter } from "react-router-dom";

// smoke test see if it renders
test("see if it renders", () => {
  render(
    <MemoryRouter>
      <App />
    </MemoryRouter>
  );
});
test("see if it renders", () => {
  const { asFragment } = render(
    <MemoryRouter>
      <App />
    </MemoryRouter>
  );
  expect(asFragment()).toMatchSnapshot();
});
