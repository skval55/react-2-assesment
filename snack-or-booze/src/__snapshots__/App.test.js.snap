// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`see if it renders 1`] = `
<DocumentFragment>
  <p>
    Loading â€¦
  </p>
</DocumentFragment>
`;
